svc/pkg/cluster/worker/src/workers/server_install/install_scripts/components/traefik.rs

```diff
@@ -282,3 +294,32 @@ pub async fn gg_static_config() -> GlobalResult<String> {
 
 	Ok(config)
 }
+
+pub fn gg_dynamic_config(datacenter_id: Uuid) -> GlobalResult<String> {
+	let domain_job = unwrap!(util::env::domain_job(), "dns not enabled");
+
+	let main = format!("{datacenter_id}.{domain_job}");
+
+	Ok(formatdoc!(
```

Comment:
nit: usually group *.tls.domain under the main block instead of alternating blocks. thx toml very cool

Reply:
@MasterPtato oh jeez i'm blind i misread the http.services and http.routers on this comment